actions/compatibility/entrypoint/main.go

```diff
@@ -0,0 +1,296 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+
+	msemver "github.com/Masterminds/semver"
+	"github.com/blang/semver"
+	"github.com/cloudfoundry/buildpacks-ci/tasks/cnb/helpers"
+	"github.com/mitchellh/mapstructure"
+	_ "github.com/pkg/errors"
```

Comment:
I have no clue why this is underscore imported. If anyone else does that would be very interesting to hear about!